src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -58,27 +58,27 @@ def __init__(
             topic = settings.KAFKA_EVENTS_SUBSCRIPTIONS_RESULTS
         self.topic = topic
         cluster_name = settings.KAFKA_TOPICS[topic]["cluster"]
-        self.bootstrap_servers = settings.KAFKA_CLUSTERS[cluster_name]["bootstrap.servers"]
         self.commit_batch_size = commit_batch_size
         self.initial_offset_reset = initial_offset_reset
         self.offsets = {}
         self.consumer = None
+        self.cluster_options = kafka_config.get_kafka_consumer_cluster_options(
+            cluster_name,
+            {
+                "group.id": self.group_id,
+                "session.timeout.ms": 6000,
+                "auto.offset.reset": self.initial_offset_reset,
+                "enable.auto.commit": "false",
```

Comment:
@wedamija, do these properties (self.group_id and self.initial_offset_reset) ever get modified on an instance of this class? Before this PR the config dictionary was created every time we executed `run`. Now this is created only once in the constructor, so if those attributes are mutable this may be an issue.

Reply:
I've refactored with confidence as I saw that the only place this class is instantiated is within a click cli command.
But I might have missed access to these fields, if they are mutable.

Reply:
ping @wedamija :)